cmd/plugin/install.go

```diff
@@ -6,17 +6,19 @@ import (
 
 	"github.com/odpf/optimus/cmd/logger"
 	"github.com/odpf/optimus/config"
+	"github.com/odpf/optimus/plugin"
 )
 
 type installCommand struct {
-	logger       log.Logger
-	serverConfig *config.ServerConfig
+	logger         log.Logger
+	serverConfig   *config.ServerConfig
+	configFilePath string `default:"config.yaml"`
 }
 
 // NewInstallCommand initializes plugin install command
-func NewInstallCommand(serverConfig *config.ServerConfig) *cobra.Command {
+func NewInstallCommand() *cobra.Command {
 	install := &installCommand{
-		serverConfig: serverConfig,
+		serverConfig: &config.ServerConfig{},
```

Comment:
This should use the default value for pointer, we are overriding this value in preRun.